hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/launcher/ContainerLauncherImpl.java

```diff
@@ -379,27 +379,38 @@ class EventProcessor implements Runnable {
 
     @Override
     public void run() {
-      LOG.info("Processing the event " + event.toString());
+      LOG.info("Processing the event {}", event.toString());
```

Comment:
You don't need the toString() as logger does it for you.

Reply:
Thanks. Made the change.